foqus_lib/framework/graph/OptGraphOptim.py

```diff
@@ -14,9 +14,6 @@
 #################################################################################
 import numpy
 
-from foqus_lib.framework.graph.edge import *
-from foqus_lib.framework.graph.node import *
```

Comment:
Yeah, when I removed then, pylint didn't find any problems.

It might be the case that these were "transitive" imports, in that another file did a wildcard import of this OptGraphOptim module to pick up whatever was in these modules.  But, I tried to make sure any imports were not transitive but came directly from the module that defined the object needed.